src/common/components/composite/sections/DashboardSection/DashboardColumn.tsx

```diff
@@ -29,3 +31,5 @@ const DashboardColumn: FC<ContextSectionColumnProps> = ({ children }) => {
 };
 
 export default DashboardColumn;
+
+const isReactElement = (element: unknown): element is ReactElement => !!(element as ReactElement).type;
```

Comment:
it's just a styling comment, but I think this should go at the top.
I am used to find the default exported element at the bottom, and all the helper functions over it, or in a separate utils file